(MAINT) Updated CHANGELOG.md for tk-jetty9 1.3.0 release

camlow325 opened the pull request:
This commit updates the CHANGELOG.md for the tk-jetty9 1.3.0 release.

cprice404 commented:
+1

fpringvaldsen commented:
:+1:

fpringvaldsen commented:
I'll merge once Travis passes

dankreek commented on CHANGELOG.md:
```diff
@@ -1,3 +1,37 @@
+This is a "feature" and security release.
+
+* [TK-178](https://tickets.puppetlabs.com/browse/TK-178) Upgrade to version
+  v9.2.10.  This includes changes made in the v9.2.9 to address a critical
```
Maybe it's obvious, but we may want to explicitly state we upgraded to "Jetty v9.2.10"

dankreek commented:
I'm good with this unless you want to this to read as pedantically as I do. :+1:

camlow325 commented:
Thanks, @dankreek.  I updated the text to hopefully make the fact that we're upgrading to Jetty v9.2.10 a little more obvious.  I just replaced the prior commit with e43f936.
